components/writers/bi-tools/looker/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Looker
+permalink: /components/writers/bi-tools/looker/
+---
+
+* TOC
+{:toc}
+
+This writer pushes data to your Looker instance. It currently uses Snowflake as backend. In broad terms it works by creating a Connection in Looker which a new LookML project can be created. It formats the columns in a way that allows automatic generation of data relations when the project is created. 
+
+## Setup
```

Comment:
dela to divny toc, pryc s tim - jen ## Configuration

Reply:
Fixed